dev-docs/modules/jwplayerRtdProvider.md

```diff
@@ -116,11 +107,15 @@ Each bid for which targeting information was found will conform to the following
     adUnitCode: 'xyz',
     bidId: 'abc',
     ...,
-    jwTargeting: {
-      segments: ['123', '456'],
-      content: {
-        id: 'jw_abc123'
-      }
+    fpd: {
+      context: {
+         data: {
+           jwTargeting: {
+             segments: ['123', '456'],
+             content: {
+               id: 'jw_abc123'
+           }
+        }
```

Comment:
this is wrong, we are not adding the targeting information to `fpd`, we are adding it on the bid object. We read from `adUnit.fpd` and write to bid.